src/node/internal/internal_utils.ts

```diff
@@ -26,16 +26,17 @@
 // OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-import { default as bufferUtil } from 'node-internal:buffer';
 import type { Encoding } from 'node-internal:buffer';
-import { validateFunction } from 'node-internal:validators';
+import { default as bufferUtil } from 'node-internal:buffer';
 import { ERR_FALSY_VALUE_REJECTION } from 'node-internal:internal_errors';
+import { validateFunction } from 'node-internal:validators';
 
 const { UTF8, UTF16LE, HEX, ASCII, BASE64, BASE64URL, LATIN1 } = bufferUtil;
 
 export function normalizeEncoding(enc?: string): Encoding | undefined {
   if (
     enc == null ||
+    enc === undefined ||
```

Comment:
Line 38 already checks for null and undefined. This line is not needed.

Reply:
Oh, fixed, I missed the `==` thing, thanks